styleguide/_themes/derek/scss/components/ticker.scss

```diff
@@ -0,0 +1,38 @@
+.ticker {
+  background-color: $rackspace-red;
+  width: 100%;
```

Comment:
Is `width: 100%;` entirely necessary? 
If this is happening out outside of a container, isn't already set at full width? (Just a question.  I haven't tested it).